lt/observability/src/test/java/com/google/cloud/teleport/lt/spanner/GenericTemplateLTObserver.java

```diff
@@ -0,0 +1,120 @@
+package com.google.cloud.teleport.lt.spanner;
+
+import com.google.auth.Credentials;
+import com.google.cloud.teleport.lt.AssertAll;
+import com.google.cloud.teleport.lt.dataset.bigquery.BigQueryPerfDataset;
+import com.google.cloud.teleport.lt.dataset.bigquery.BigQueryPerfDatasetFetcher;
+import com.google.cloud.teleport.lt.dataset.bigquery.PerfResultRow;
+import com.google.cloud.teleport.lt.rules.DetectMetricDeviationConditionCheck;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.beam.it.common.TestProperties;
+import org.apache.beam.it.gcp.bigquery.BigQueryResourceManager;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+import org.junit.runners.model.MultipleFailureException;
+
+/** Load test observer for all the spanner templates. */
+@RunWith(Parameterized.class)
+public class GenericTemplateLTObserver {
+
+  private static final Credentials CREDENTIALS = TestProperties.googleCredentials();
+
+  private static BigQueryResourceManager bigQueryResourceManager;
+
+  @Parameters
+  public static Collection<String[]> getParameters() {
+    String projectId = TestProperties.exportProject();
+    bigQueryResourceManager =
+        BigQueryResourceManager.builder("Load-test-analyzer", projectId, CREDENTIALS)
+            .setDatasetId(TestProperties.exportDataset())
+            .build();
+    // *DO NOT CALL cleanup on this bigQueryResourceManager.* It would delete the tables in the
```

Comment:
can we disable this via FGAC to prevent this possibility ?

Reply:
I will check. Basically the account needs write access to a table but not delete table access. I'll check if this is possible.